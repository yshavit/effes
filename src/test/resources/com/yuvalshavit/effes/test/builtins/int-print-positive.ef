getNum -> IntValue: return 27

main:
  print getNum
