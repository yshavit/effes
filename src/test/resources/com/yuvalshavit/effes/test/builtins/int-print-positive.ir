def getNum () -> IntValue:
  ReturnStatement
    IntLiteral
      27
