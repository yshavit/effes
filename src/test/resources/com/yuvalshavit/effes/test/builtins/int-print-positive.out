Your argument was: 27
>> null
